administration/configuring-fluent-bit/multiline-parsing.md

```diff
@@ -180,4 +180,104 @@ The multiline parser is a very powerful feature, but it has some limitations tha
 
 * The multiline parser is not affected by the `buffer_max_size` configuration option, allowing the composed log record to grow beyond this size.
 Hence, the `skip_long_lines` option will not be applied to multiline messages. 
-* It is not possible to get the time key from the body of the multiline message. However, it can be extracted and set as a new key by using a filter. 
\ No newline at end of file
+* It is not possible to get the time key from the body of the multiline message. However, it can be extracted and set as a new key by using a filter.
+
+## Get structured data from multiline message
+
+Fluent-bit supports `/pat/m` option. It allows `.` matches a new line. It is useful to parse multiline log.
+
+The following example is to get `date` and `message` from concatenated log.
+
+Example files content:
+
+{% tabs %}
+{% tab title="fluent-bit.conf" %}
+This is the primary Fluent Bit configuration file. It includes the `parsers_multiline.conf` and tails the file `test.log` by applying the multiline parser `multiline-regex-test`. It also parses concatenated log by applying parser `named-capture-test`. Then it sends the processing to the standard output.
+
+```
+[SERVICE]
+    flush        1
+    log_level    info
+    parsers_file parsers_multiline.conf
+
+[INPUT]
+    name             tail
+    path             test.log
+    read_from_head   true
+    multiline.parser multiline-regex-test
+
+[FILTER]
+    name             parser
+    match            *
+    key_name         log
+    parser           named-capture-test
+
+[OUTPUT]
+    name             stdout
+    match            *
+```
+{% endtab %}
+
+{% tab title="parsers_multiline.conf" %}
+This second file defines a multiline parser for the example.
+
+```
+[MULTILINE_PARSER]
+    name          multiline-regex-test
+    type          regex
+    flush_timeout 1000
+    #
+    # Regex rules for multiline parsing
+    # ---------------------------------
+    #
+    # configuration hints:
+    #
+    #  - first state always has the name: start_state
+    #  - every field in the rule must be inside double quotes
+    #
+    # rules |   state name  | regex pattern                  | next state
+    # ------|---------------|--------------------------------------------
+    rule      "start_state"   "/(Dec \d+ \d+\:\d+\:\d+)(.*)/"  "cont"
```

Comment:
It'd be great to make this `(?[a-zA-Z]+ \d+ \d+\:\d+\:\d+)` as well so the example isn't locked on a December log line. 

Easy follow up though :)

Reply:
Thank you for it.
I sent another PR https://github.com/fluent/fluent-bit-docs/pull/994